### ISSUE TITLE
Update and Deploy with SVN

### ISSUE THREAD

**Issue by rijad-cz** [rocketeers/rocketeer#448]
Hi, 

I use Rockeeter with SVN but I recognized, that it works not correctly. When I run "rockeeter update" or "rocketeer deploy" it seems to work correctly, but on the server remain the same SVN revision as before.

I found the problem, It's not related with the deploy strategy, but in the SVN class  Rocketeer\Scm\Svn is a little mistake. 

```php
public function reset()
{
		$command = sprintf('status -q | grep -v \'^[~XI ]\' | awk \'{print $2;}\' | xargs %s revert', $this->binary);

		return $this->getCommand($command);
}
```

"%s revert" this is the mistake, correct should be "%s revert .". SVN revert need path as argument 

example from console in some SVN working copy

➜  trunk  svn revert
svn: E205001: Try 'svn help revert' for more information
svn: E205001: Not enough arguments provided

➜  trunk  svn revert .
➜  trunk

Could you fix it?

Thank you.

**Comment by Anahkiasen** [comment]
I think that's been fixed in 2.0.6 no? In 32fadfd43290f9d4253048c5f2f469cae1f50e90 ? Or would that not be what you meant? I don't use SVN personally, the SVN class has been contributed by users so I'm not sure what works or doesn't work to be honest, nor do I have an SVN repository under hand to test with :/

**Comment by rijad-cz** [comment]
Yes, you are right it was fixed in 2.0.6. Sorry I had 2.0.5. It can be closed. Thank you
